git/week1/countries.txt

```diff
@@ -3,4 +3,5 @@ spain
 UK
 USA
 France
-Seychelles
```

Comment:
i don't know why this got deleted and added back below, i. haven't done anything regarding this text.

Reply:
That is a git thing, i also don't understand why it happens :/